portfolio/src/main/webapp/script.js

```diff
@@ -47,18 +47,34 @@ function getComments() {
 function createListElement(text) {
   const liElement = document.createElement('li');
   liElement.className = 'comment';
+
   const nameElement = document.createElement('span');
-  nameElement.innerText = text.name;
+  nameElement.innerText = text.name + " ";
+
   const dateElement = document.createElement('span');
   const date = new Date(text.timestamp);
   const dateTime = " - " + date.toDateString() + " " + date.getHours() + ":" + date.getMinutes();
   dateElement.innerText = dateTime;
+
   const textElement = document.createElement('p');
   textElement.innerText = text.text;
 
+  const sentimentScoreElement = document.createElement('img');
+  const score = Math.round(text.sentimentScore * 100) / 100
```

Comment:
What is this line for? Why not just use text.sentimentScore directly? (Genuine question! I'm not saying this is wrong, I'm just saying I don't understand it.)

Reply:
Ah, after reading the code below, I see that you're doing it to round score to 2 digits. I'm still not sure what the advantage of doing that is, so please add a comment explaining to the reader (including your future self!) why this is a good idea.